README.md

```diff
@@ -33,6 +33,26 @@ To directly access the **Getting Started Guide** for supported hardware platform
 
 For detailed documentation on FreeRTOS, refer to the [FreeRTOS User Guide](https://aws.amazon.com/documentation/freertos).
 
+### AWS Collection of Metrics
+
+AWS collects usage metrics indicating the operating system, and MQTT client information of use to AWS IoT from the MQTT mutual auth demo by sending a specially formatted string in the username field of the MQTT CONNECT packet. These metrics help AWS IoT improve security and provide better technical support. Providing these metrics is optional for users, and can be disabled by updating the `OS_NAME`, `OS_VERSION` and `MQTT_LIB` configuration macros in the `demos/include/aws_iot_metrics.h` file.
```

Comment:
```suggestion
MQTT mutual auth demo reports metrics about the operating system, and MQTT client library used to AWS IoT by sending a specially formatted string in the username field of the MQTT CONNECT packet. These metrics help AWS IoT improve security and provide better technical support. Providing these metrics is optional for users, and these can be disabled by updating the following configuration macros in the `demos/include/aws_iot_metrics.h` file:
* #define AWS_IOT_METRICS_STRING    NULL
* #define AWS_IOT_METRICS_STRING_LENGTH 0U
```

Reply:
It is not the MQTT mutual auth demo but the Defender, Shadow and Jobs demos that report the metrics with this PR's changes. I will update the text accordingly.